install/kubernetes/cilium/values.yaml

```diff
@@ -1777,6 +1778,30 @@ tls:
     # -- Generated certificates validity duration in days. This will be used for auto generated CA.
     certValidityDuration: 1095
 
+  # -- Configure the CA trust bundle used for the validation of the certificates
+  # leveraged by hubble and clustermesh. When enabled, it overrides the content of the
+  # 'ca.crt' field of the respective certificates, allowing for CA rotation with no down-time.
+  caBundle:
+    # -- Enable the use of the CA trust bundle.
+    enabled: false
```

Comment:
Maybe instead of an enabled field, it's enabled if `caBundle.name` is set, and we default `caBundle.name` to empty.

Reply:
I personally feel that the explicit `enabled` field is easier to understand, and the same pattern is also widely adopted in the rest of the values file. Specifying a non-empty name has also the advantage of suggesting a consistent name across installations.